content/package.json

```diff
@@ -8,14 +8,10 @@
     "build": "node ./build.js",
     "clean": "rimraf lib",
     "dev": "yarn build",
-    "prettier": "prettier --write . --ignore-path=./.eslintignore",
-    "prettier:check": "prettier . --check --ignore-path=./.eslintignore",
+    "prettier": "prettier --write . --ignore-path=../.eslintignore",
```

Comment:
eslintignore file does not exist in the content folder, updated file path so it uses the file in root. not we are skipping the content folder in root, not sure if this is desired